master/buildbot/status/web/waterfall.py

```diff
@@ -183,8 +183,8 @@ def getBox(self, req):
                 url = None
             cxt['logs'].append(dict(name=name, url=url))
 
-        for name, target in self.original.getURLs().items():
-            cxt['urls'].append(dict(link=target,name=name))
+        for url in self.original.getURLs():
+            cxt['urls'].append(dict(link=url["url"],name=url["name"]))
```

Comment:
There is might be a good reason to iterate over the list of dictionary objects to only extract their properties, rename some of them and convert back to dictionary objects instead of copying the whole list. Can we have 'url' everywhere?